src/statusnotifieritem/statusnotifieritem.cpp

```diff
@@ -287,7 +287,8 @@ IconPixmapList StatusNotifierItem::iconToPixmapList(const QIcon& icon)
     IconPixmapList pixmapList;
 
     // long live KDE!
-    for (const QSize &size : icon.availableSizes())
+    const QList<QSize> sizes = icon.availableSizes();
+    for (const QSize &size : const_cast<QList<QSize>&>(sizes))
```

Comment:
@palinek Of course no. It's missing an `const`.
IMO We should use `qAsConst()` as soon as we can. In that sense we should not use something like this:
``` cpp
for (QChar ch : qAsConst(funcReturningQString()))
    process(ch); // ERROR: ch is copied from deleted memory
```